modules/extensions-xml-support/src/test/java/org/mule/runtime/extension/internal/loader/validation/DefaultModelValidatorTestCase.java

```diff
@@ -220,17 +221,17 @@ private ExtensionModel getExtensionModelFrom(String modulePath, Set<ExtensionMod
   private Set<ExtensionModel> getDependencyExtensions() {
     ExtensionModel petstore = loadExtension(PetStoreConnector.class, emptySet());
     ExtensionModel heisenberg = loadExtension(HeisenbergExtension.class, emptySet());
-    return ImmutableSet.<ExtensionModel>builder().add(petstore, heisenberg).build();
+    return ImmutableSet.<ExtensionModel>builder().add(petstore, heisenberg, getExtensionModel()).build();
   }
 
-  private ExtensionModel loadExtension(Class extension, Set<ExtensionModel> deps) {
+  private ExtensionModel loadExtension(Class extension, Set<ExtensionModel> extensions) {
```

Comment:
should this be called depedencyExtensions or something like that? The intent of the parameter is now lost